DcShare/Controller/ShareViewController.swift

```diff
@@ -31,11 +31,14 @@ class ShareViewController: SLComposeServiceViewController {
         }
     }
 
+    lazy var dbHelper: DatabaseHelper = {
+       return DatabaseHelper(dcContext: dcContext)
+    }()
+
     let logger = SimpleLogger()
-    let dcContext = DcContext.shared
+    let dcContext: DcContext = DcContext()
```

Comment:
dcContext is now unusable until presentationAnimationDidFinish() is processed, just saying, probably okay.

Reply:
yes it is the main entry point for the extension